packages/presets/src/ai/actions/edgeless-response.ts

```diff
@@ -272,12 +272,20 @@ export const responses: {
     aiPanel.hide();
 
     const edgelessRoot = getEdgelessRootFromEditor(host);
-    const { left, top, height } = selectionRect;
+    const { left: x, top, height } = selectionRect;
+    const y = top + height + 20;
+    const data = ctx.get();
+    const w = (data['width'] as number) || 400;
+    const h = (data['height'] as number) || 200;
```

Comment:
I feel 400x200 is a bit small, most results won't fit, maybe something bigger like 800x600